### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by nomoreboredom**
As a novice in webpack it took a while for me to figure out how to run the examples :) So I wanted to add Instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+## To Run An Example
+
+Clone or download this repo.
+
+Navigate to an example's root folder, then run
+
+```
+npm install 
+```
+To build the project, run
```
Comment (tshaddix) [inline README.md]: Can we add a newline here to stay consistent?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+
+```
+npm install 
+```
+To build the project, run 
+
+```
+gulp
+```
+And webpack bundle will be created. You can now use new created content folder to upload to Chrome.
```
Comment (tshaddix) [inline README.md]: Ditto here. Can we add a newline?

Comment (tshaddix) [inline README.md]: Maybe we should say the name of the build folder here? Something like:

> In the root project directory, you will find a `build` folder. To install the extension in chrome, go to [chrome://extensions](chrome://extensions) and "Load unpacked extension...". Select the `build` directory and you're on you're way!

**Comment by nomoreboredom** [comment]
Hi @tshaddix,

I have implemented the changes you requested. This PR gets automatically updated with edits right? Excuse my GitHub ignorance :)

**Comment by vhmth** [comment]
@nomoreboredom yup! PRs are tied to a branch with your changes and a base branch where your changes will be merged into once accepted. So whenever you push code to your branch, it updates here. :-)

**Pull request merged**